test/functional/helpers/spectrum3.js

```diff
@@ -211,23 +190,35 @@ const componentWrappers = {
         popoverMenuSelector
       ),
       async enterSearch(text) {
-        await switchToIframe();
         await t.typeText(selector, text).pressKey("enter");
       },
       async clear() {
-        await switchToIframe();
         await t.selectText(selector).pressKey("delete");
       },
       async scrollToTop() {
-        await switchToIframe();
         await t.scroll(popoverMenuSelector, 0, 0);
       },
-      // When the combobox loads pages of data when scrolling this
-      // will keep scrolling until the end of the last page.
-      async scrollToBottom() {
-        await switchToIframe();
-        await t.scrollIntoView(
-          popoverMenuSelector.find(menuItemCssSelector).nth(-1)
+      // When the combobox loads pages of data when scrolling, this
+      // will keep scrolling until the the item is reached.
+      async scrollDownToItem(label) {
+        for (let i = 0; i < 10; i += 1) {
+          if (
+            // eslint-disable-next-line no-await-in-loop
+            await popoverMenuSelector
+              .find(menuItemCssSelector)
+              .withExactText(label).exists
+          ) {
+            return;
+          }
+
+          // eslint-disable-next-line no-await-in-loop
+          await t.scrollIntoView(
+            popoverMenuSelector.find(menuItemCssSelector).nth(-1)
+          );
+        }
+
+        throw new Error(
+          `Option with label ${label} does not exist while scrolling down when it is expected to exist.`
```

Comment:
When the extension view was running inside the iframe, something in TestCafe's logic was continuing to scroll until reaching the last page. I was a bit surprised this was actually happening, though it wasn't guaranteed in the docs anywhere. Now that the extension view is running outside the iframe, due to changes in timing I imagine, it will typically scroll a couple pages, but wouldn't continue scrolling until the end of the last page. These change take a more predictable approach.